src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version() -> VersionNumber
+
+### Purpose ###
```

Comment:
Let's follow the documentation guidelines in the Julia docs and omit this `### Purpose ###` line.